sdk/cosmos/azure-cosmos-spark_3-1_2-12/docs/quick-start.md

```diff
@@ -11,8 +11,8 @@ You can use any other Spark 3.1.1 spark offering as well, also you should be abl
 - An active Azure account. If you don't have one, you can sign up for a 
   [free account](https://azure.microsoft.com/try/cosmosdb/).
   Alternatively, you can use the
-  [use Azure Cosmos DB Emulator](https://docs.microsoft.com/azure/cosmos-db/local-emulator) for development and testing.
-- [Azure Databricks](https://docs.microsoft.com/azure/databricks/release-notes/runtime/8.0)
+  [use Azure Cosmos DB Emulator](local-emulator.md) for development and testing.
```

Comment:
This will break the github changelog page. The link currently works so why does this require removing the URI?

Reply:
The CI is actually failing because malformed links are identified https://dev.azure.com/azure-sdk/public/_build/results?buildId=1043988&view=logs&jobId=b70e5e73-bbb6-5567-0939-8415943fadb9&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=d34cfe88-f3e1-5ca1-ec50-18a5cea1099a

Reply:
+1 this breaks the links on Github - which is the primary target audience.

Reply:
Docs require us to use relative links because absolute links don't work on air gapped/JEDI environments. in the current form, it breaks the docs build. it's currently shown as a warning, and will eventually turn into an error. 

I also raised a bug to exclude referenced content from build validation. However, that's not possible because other services need validation and my bug was closed by saying that's by design.